.env

```diff
@@ -35,9 +35,6 @@ PG_APPLICATION_NAME=stacks-blockchain-api
 # See https://node-postgres.com/api/pool
 # PG_CONNECTION_POOL_MAX=10
 
-# Enable to have stacks-node events streamed to a file while the application is running
-# STACKS_EXPORT_EVENTS_FILE=/tmp/stacks-events.tsv
```

Comment:
I think we should keep this ENV var here (or documented somewhere else) since it's still used during event export or event replay.

Reply:
Actually, it's no longer used anywhere 👍 
That ENV var only controlled the file path of the live TSV, so feel free to remove it from the pod ENV settings